### DISCUSSION TITLE
Integration with yamllint

### DISCUSSION THREAD

**Discussion by tteguayco** [redhat-developer/vscode-yaml#989] [Q&A]
[yamllint](https://github.com/adrienverge/yamllint) is well-known linter for YAML files.

Is it possible to use [yamllint config files](https://yamllint.readthedocs.io/en/stable/configuration.html) with this VSCode extension?

**Comment by gorkem** [comment]
No it is not possible. I do not think there is full parity with yamliint at this time either

**Comment by bdsoha** [comment]
That is very unfortunate.
